contracts/MarginBase.sol

```diff
@@ -706,4 +714,18 @@ contract MarginBase is MinimalProxyable, IMarginBase, OpsReady {
     function _abs(int256 x) internal pure returns (uint256) {
         return uint256(x < 0 ? -x : x);
     }
+
+    /// @notice added to support recovering trapped erc20 tokens
+    /// @param tokenAddress: address of token to be recovered
+    /// @param tokenAmount: amount of token to be recovered
+    function rescueERC20(address tokenAddress, uint256 tokenAmount)
+        external
+        onlyOwner
```

Comment:
can we add a test to make sure it's never callable by non owner. just want to make sure this never gets removed by accident?

Reply:
Added ✅